client/js/components/home.jsx

```diff
@@ -87,7 +87,13 @@ class Home extends React.Component {
       this.resetHierarchy(bank, map);
     }
     let itemChecked = Object.assign({}, this.state.itemChecked, map);
-    this.setState({itemChecked});
+    this.setState({
+      itemChecked,
+      isOpen: false,
+      assessmentClicked: {},
+      currentBankId: null,
+      nOfM: null
+    });
```

Comment:
This is the easiest way to do this. The alternative is moving some of this (isOpen, assessmentClicked, currentBankId) into the store, and then clearing it when we clear snippet.